src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,32 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import cn from 'classnames';
+import { User, UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo,
+};
+
+export interface Todo {
+  id: number,
+  title: string,
+  completed: boolean,
+  userId: number,
+  user?: User,
+}
+
+export const TodoInfo:React.FC<Props> = ({ todo }) => {
+  const { title, completed, user } = todo;
+
+  return (
+    <article
+      data-id={todo.id}
+      className={cn('TodoInfo', { 'Todo-Info': completed })}
```

Comment:
```suggestion
      className={cn('TodoInfo', { 'The_class_from_tests': completed })}
```
<img width="494" alt="Screenshot 2024-02-06 at 16 00 10" src="https://github.com/mate-academy/react_add-todo-form/assets/135232001/78a5fd2e-977a-4808-878b-7d51eeb31099">